en/components/hierarchicalGrid/hierarchical_grid_load_on_demand.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Hierarchical Grid load on demand - Native Angular | Ignite UI for Angular
+_description: The Ignite UI for Angular Hierarchical Grid provides the necessary tools to load data on demand for each child grid that is expanded. That way the volume of data would be greatly reduced and can be retrieved only when the user needs it.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Tree Grid component, Angular Tree Grid control, Angular Tree Grid component, Angular High Performance Tree Grid, Hierarchical Grid 
+---
+
+## Hierarchical Grid Load On Demand
```

Comment:
All grid/treegrid/hierarchicalgrid feature topics begin with header 3 (**###**) instead of header 2 (**##**), so for consistency it would be nice to have it that way here as well.